tools/make/image.mk

```diff
@@ -9,6 +9,7 @@ include tools/make/env.mk
 IMAGES_DIR ?= $(wildcard ${ROOT_DIR}tools/docker/*)
 # Determine images names by stripping out the dir names
 IMAGES ?= api-testing
+IMAGE_NAME ?= ${REGISTRY}/${REGISTRY_NAMESPACE}/${IMAGES}/${TAG}
```

Comment:
`${REGISTRY}/${REGISTRY_NAMESPACE}/${IMAGES}/${TAG}` should be `${REGISTRY}/${REGISTRY_NAMESPACE}/${IMAGES}:${TAG}`

Reply:
![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/06cc40e3-d801-4fdc-a0a9-66fa6e46d3b3)

Reply:
fixed.